_data/authors.yml

```diff
@@ -20,4 +20,14 @@ tjdraper:
     TJ is a independent web developer and owner of [BuzzingPixel, LLC](https://buzzingpixel.com/) in Spring Hill, TN. He is also a veteran podcaster — having survived 150 episodes as host of [The MovieByte Podcast](https://moviebyte.com/mbpodcast). Now TJ hosts a new show about appreciating cinema called [Retake](https://nightowl.fm/retake) on the podcast network he launched with [Joe Darnell](https://nightowl.fm/people/joedarnell), [Night Owl](https://nightowl.fm/).
   author_path: tjdraper
   gravatar: 979fa599e691b040164ca31d848a9cf9
-
+keatontaylor:
+  display_name: Keaton Taylor
+  title: Contributing Writer
+  twitter: keaton_taylor
+  github: keaton-taylor
+  web: www.keaton-taylor.com
+  email: keatontaylor@me.com
+  bio: >
+   Keaton is a designer and frontender working on editorial products at [Apartment Therapy Media](http://www.apartmenttherapy.com) from his home in sunny Southwest Missouri. Keaton loves his lovely wife and three children, craft beer, mediocre tv and Falltime. 
```

Comment:
"Falltime" is technically not a thing. Which makes no sense considering that [every other season has one](http://english.stackexchange.com/questions/185221/why-is-there-no-autumntime-or-falltime). I say we keep it and start the "Falltime" trend.